### ISSUE TITLE
Check boxes selectable across entire widget, leading to accidental checks.

### ISSUE THREAD

**Issue by Nick2253** [hexparrot/mineos-node#281]
Check boxes, in particular in the "Server Status" window, are selectable across the entire widget.  This means that clicking in the empty whitespace on the right side of a widget can accidentally check or uncheck the boxes.

For example, if I'm in a second window, and I click back to my browser to interact with MineOS, if I click in that whitespace, I can accidentally trigger changes to my server config.

I suggest making the checkbox "hitbox" only the same length as the text, if not just the checkbox.

**Comment by hexparrot** [comment]
The problem here is that the hitbox expands is a _feature_ of the responsive design template I've purchased.  That is to say, in order to make the additional whitespace (that doesn't feel like the label, but in fact is), would mean I would have to micromanage the labels' widths themselves.

This isn't an insurmountable problem, but _fixing_ this behavior (fixed-width rather than auto expand to fill) also means structurally changing the URL with placeholders. This could impact mobile displaying, different languages, and generally goes against the philosophy of the template itself.

If you want an example of my inexpertise with the template, see what happens already when the display width is lessened:  https://imgur.com/a/AFgmlyO

In short, I'm not against such an adjustment, but any adjustment has to be a strict improvement over what's currently there, and I --personally--am not sure I know how to modify it better than to have expand-full-container behavior.

**Comment by ScottyPcGuy-03** [comment]
I currently do not have access to my other github account, or I would submit a pull request. The solution is actually pretty straightforward and requires just two changes to the bootstrap css:

Add `clear: right` to the below section

`.radio,.checkbox {
    min-height: 20px;
    padding-left: 20px;
    clear: right;
}`

and `width: fit-content` to

`label {
    display: block;
    margin-bottom: 5px;
    width: fit-content;
}`

in `bootstrap.min.css`.
